src/winter_onboarding_2021/fleet_management_service/session.clj

```diff
@@ -0,0 +1,17 @@
+(ns winter-onboarding-2021.fleet-management-service.session
+  (:require [winter-onboarding-2021.fleet-management-service.utils :as utils]
+            [winter-onboarding-2021.fleet-management-service.db.session :as session-db]))
+
+(defn new [user-id]
+  (let [session-id (utils/uuid)]
+    (session-db/insert session-id user-id)
+    session-id))
+
+(defn lookup [session-id]
+  (session-db/lookup session-id))
+
+(defn delete [session-id]
+  (session-db/delete session-id))
+
+(defn join-user-with-session [session-id]
```

Comment:
Can we call this something else? Let's stick with a noun if possible. I'd suggest `user-session` as a name of the map that has both user & session information.